manifests/domain.pp

```diff
@@ -1,21 +1,44 @@
-# == Define: cvmfs::domain
-define cvmfs::domain ($cvmfs_quota_limit = undef,
-  $cvmfs_server_url = undef,
-  $cvmfs_http_proxy = undef,
-  $cvmfs_timeout = undef,
-  $cvmfs_timeout_direct = undef,
-  $cvmfs_nfiles = undef,
-  $cvmfs_public_key = undef,
-  $cvmfs_force_singing = undef,
-  $cvmfs_max_ttl = undef,
-  $cvmfs_env_variables = undef,
-  $cvmfs_use_geoapi = undef,
-  $cvmfs_follow_redirects = undef,
-  Optional[String] $cvmfs_external_fallback_proxy = undef,
-  Optional[String] $cvmfs_external_http_proxy = undef,
-  Optional[Integer] $cvmfs_external_timeout = undef,
+# @summary Configure a a domain of cvmfs repositories
+# @example Add the example.org domain
+#  cvmfs::domain{'example.org':
+#    cvmfs_server_url = 'http://example.org/cvmfs/@fqrn@',
+#  }
+# @param cvmfs_quota_limit Per mount quota limit, not relavent to shared cache. Sets cvmfs_quota_limit
+# @param cvmfs_server_url Stratum 1 list, typically `;` delimited. Sets CVMFS_SERVER_URL parameter.
+# @param cvmfs_http_proxy List of http proxies to use. Sets CVMFS_PROXY_LIST parameter.
+# @param cvmfs_timeout Sets CVMFS_HTTP_TIMEOUT parameter
+# @param cvmfs_timeout_direct Sets CVMFS_HTTP_TIMEOUT_DIRECT
+# @param cvmfs_nfiles Number of open files permitted on the OS. Sets CVMFS_NFILES
+# @param cvmfs_max_ttl Maximum effective TTL in seconds for DNS queries of proxy server names. Sets CVMFS_MAX_TTL
+# @param cvmfs_env_variables Sets per repo environments variables for magic links.
+# @param cvmfs_use_geoapi Set CVMFS_MAX_GEOAPI
+# @param cvmfs_follow_redirects Sets CVMFS_FOLLOW_REDIRECTS.
+# @param cvmfs_external_fallback_proxy Sets CVMFS_EXTERNAL_FALLBACK_PROXY
+# @param cvmfs_external_http_proxy Sets CVMFS_EXTERNAL_HTTP_PROXY
+# @param cvmfs_external_timeout Sets CVMFS_EXTERNAL_TIMEOUT
+# @param cvmfs_external_timeout_direct Sets CVMFS_EXTERNAL_TIMEOUT_DIRECT
+# @param cvmfs_external_url Sets CVMFS_EXTERNAL_URL
+# @param cvmfs_public_key Specify repository signing key
+# @param cvmfs_force_singing Force signing, deprecated.
+#
+define cvmfs::domain (
+  Optional[Integer] $cvmfs_quota_limit             = undef,
```

Comment:
It seems that `cvmfs_quota_limit` is not used as is in `repo.local.erb`, it refers to `my_cvmfs_quota_limit` as passed from `config.pp`.